tests/manifests/kustomization.yaml

```diff
@@ -25,3 +25,6 @@ configMapGenerator:
     - controller_definitions.yaml
   options:
     immutable: true
+    disableNameSuffixHash: true
```

Comment:
I'm looking up what this is now. So, it basically means we'd just have a single ConfigMap in our cluster rather than one for each time the version of the ConfigMap changed, right?

Reply:
I understood that was the wanted behavior. @xdevxy do we want one configmap or multiple since the controller searches by label anyways?

Reply:
yeah, seems like the hash thing is really just related to the ConfigMap being volume mounted into the container, which we're not doing

Reply:
Since the controller searches by label, `disableNameSuffixHash: true` is not really necessary (I personally do not like hashes if not necessary). Also, we do not need `behavior: merge` at this time since we only have a single definitions file and since it was causing error. I'm thinking the error was due to also having set `immutable: true` but I'm not sure. I'm trying to see if separating the definitions in multiple files and enabling merge would make more sense.

Reply:
> I understood that was the wanted behavior. @xdevxy do we want one configmap or multiple since the controller searches by label anyways?

I'm actually fine with either.